heartbeat/docs/configuring-howto.asciidoc

```diff
@@ -1,8 +1,12 @@
 [[configuring-howto-heartbeat]]
-= Configuring {beatname_uc}
+= {beatname_uc} configuration
 
 [partintro]
 --
+++++
+<titleabbrev>Configuration</titleabbrev>
```

Comment:
It seems weird that this is a noun, when the other title abbreviations are verbs.

Reply:
OK, I'll change!